nixos/modules/services/audio/mpdscheduler.nix

```diff
@@ -0,0 +1,59 @@
+{ config, lib, pkgs, ... }:
+with lib;
+let 
+  cfg = config.services.mpdscheduler;
+in {
+  options.services.mpdscheduler = {
+    enable = mkEnableOption "mpdscheduler service";
+    host = mkOption {
```

Comment:
For non localhost values is a password required?

Reply:
Good point - it is not supported upstream yet, but I will have a look at it.